src/pylorax/api/compose.py

```diff
@@ -848,6 +848,31 @@ def compose_args(compose_type):
                                  "app_template":            None,
                                  "app_file":                None,
                                 },
+            "hyper-v":          {"make_iso":                False,
+                                 "make_disk":               True,
+                                 "make_fsimage":            False,
+                                 "make_appliance":          False,
+                                 "make_ami":                False,
+                                 "make_tar":                False,
+                                 "make_tar_disk":           True,
```

Comment:
Should be false :)

Reply:
wow, was I braindead yesterday or what. Fixed.